dev/gen_all_config_docs.sh

```diff
@@ -18,7 +18,7 @@
 
 GLUTEN_UPDATE="${GLUTEN_UPDATE:-1}" \
 mvn clean test \
-  -pl gluten-substrait -pl backends-velox -am \
+  -pl gluten-substrait -Pbackends-velox -am \
```

Comment:
should also contain `-pl backends-velox`? since the gen velox config in this module

Reply:
Updated, thanks.

Reply:
is this the right syntax? 
` -Pbackends-velox -pl backends-velox -am \`

Reply:
@zhztheplayer Right, but at least it's `-Pbackends-velox -pl backends-velox -am \`, as @zhouyuan said.

All in all, it's a small point. Thank you all!